test/features/support/voucher_cancelation.js

```diff
@@ -0,0 +1,117 @@
+const pactum = require('pactum');
+const { Given, When, Then, Before, After } = require('@cucumber/cucumber');
+const { localhost } = require('./helpers/helpers');
+
+let specVoucherCancelation;
+let serialNumberParam;
+
+const baseUrl = `${localhost}vouchers/voucherstatuscheck/{voucherserialnumber}`;
+const requestFunction = () =>
+  specVoucherCancelation.patch(baseUrl).withBody({
+    'voucherserialnumber': `${serialNumberParam}`,
+    'Gov_Stack_BB': 'string'
+  }).withPathParams('voucherserialnumber', serialNumberParam);
+
+Before(() => {
+  specVoucherCancelation = pactum.spec();
+});
+
+// Scenario: The user successfully cancels a voucher
+Given('The user wants to cancel a voucher with voucher serial number 1234', () => {
+  serialNumberParam = '1234';
+  return serialNumberParam;
+});
+
+When('The user triggers an action with a valid payload to cancel a voucher', () => {
+  requestFunction();
+});
+
+Then('The user successfully cancels the voucher', async () => {
+  await specVoucherCancelation.toss();
+  specVoucherCancelation.response().should.have.status(200);
+  specVoucherCancelation.response().should.have.jsonLike({
+    'message': 'Voucher cancelled'
+  });
+});
+
+// Scenario: The user is not able to cancel a voucher with voucher serial number 4321
```

Comment:
I think we should change this test scenario. Reason of test case failure is not voucher number but missing body. Just want to make sure if that's a convention we want to take. With regard do our recent discussion about configurable payloads it seems that including data like serial numbers in test case names is not direction we should take. 
Can we use something like: 
```suggestion
// Scenario: The user is not able to cancel a voucher without providing voucher details.
```
instead?

This is the case for other test cases in scope of this ticket as well.